crates/ruff_dev/src/main.rs

```diff
@@ -48,22 +51,29 @@ enum Command {
     PrintTokens(print_tokens::Args),
     /// Run round-trip source code generation on a given Python file.
     RoundTrip(round_trip::Args),
+    /// Format a repository twice and ensure that it looks that the first and second formatting
+    /// look the same. Same arguments as `ruff check`
+    CheckFormatterStability(check_formatter_stability::Args),
 }
 
-fn main() -> Result<()> {
+fn main() -> Result<ExitCode> {
     let args = Args::parse();
     #[allow(clippy::print_stdout)]
-    match &args.command {
-        Command::GenerateAll(args) => generate_all::main(args)?,
-        Command::GenerateJSONSchema(args) => generate_json_schema::main(args)?,
+    match args.command {
```

Comment:
i've changed this back and forth like three times :sweat_smile: i can also roll this back once again